docs/HowToFAQ/FAQLoopModifies.md

```diff
@@ -0,0 +1,58 @@
+---
+title: If I have an assertion about a class and a loop that doesn't mention (read, modify)  the class, why does dafny fail to establish the assertion after the loop?
+---
+
+## Question
+
+If I have an assertion about a class and a loop that doesn't mention (read, modify)  the class, why does dafny fail to establish the assertion after the loop?
```

Comment:
Why do we duplicate the title in every entry?

Reply:
Many do, but many also give more detail in the Question section , like the code for the problem example.